src/main/java/com/ays/backend/user/model/PhoneNumber.java

```diff
@@ -0,0 +1,20 @@
+package com.ays.backend.user.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "phoneNumbers")
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class PhoneNumber extends BaseEntity {
+
+    private Integer countryCode;
+    private Integer phoneNumber;
```

Comment:
please change the name of the property.

Reply:
I just changed the name of phoneNumber property